dataflow/flex-templates/kafka_to_bigquery/.gcloudignore

```diff
@@ -0,0 +1,7 @@
+# Ignore everything from the Cloud Build command.
```

Comment:
Copyright & Licenses - general rule - if a human creates it, then it needs a copyright & license.